backend/app/api/api_v1/routers/lookups/geo_stats.py

```diff
@@ -0,0 +1,77 @@
+from http.client import NOT_FOUND
+import logging
+from app.db.session import get_db
+from app.db.models.geography import GeoStatistics
+from fastapi import Depends, HTTPException
+from app.core.auth import get_current_active_user
+from pydantic import BaseModel
+from sqlalchemy import exc
+from .router import lookups_router
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class GeoStatsResponse(BaseModel):
+    """The response definition from /get_stats/{geography_id}"""
+
+    id: int
+    name: str
+    geography_id: int
+    legislative_process: str
+    federal: bool
+    federal_details: str
+    political_groups: str
+    global_emissions_percent: float
+    climate_risk_index: float
+    worldbank_income_group: str
+    visibility_status: str
+
+
+lookup_geo_stats_responses = {
+    404: {"description": "Statistics for Geography Id was not found"}
+}
```

Comment:
This looks a bit silly, but it would let the type checker tell us (and raise an issue) if we ever modify this responses map in the code:

```suggestion
lookup_geo_stats_responses: Mapping[int, Mapping] = {
    404: {"description": "Statistics for Geography Id was not found"}
}
```